xml/constraints-next.rng

```diff
@@ -129,12 +143,6 @@
               <ref name="attribute-roles"/>
             </attribute>
           </optional>
-          <optional>
-            <attribute name="rsc-instance"><data type="integer"/></attribute>
-          </optional>
-          <optional>
-            <attribute name="with-rsc-instance"><data type="integer"/></attribute>
-          </optional>
```

Comment:
This and the next drop are what makes this "-next"

The original idea was that the "-next" schema would get experimental features that would be promoted to the main schema once there was some experience using them. In practice that's not the workflow we follow and no one uses "-next".

Reply:
Ugh, thought I fixed this.